src/Models/LoginSession.php

```diff
@@ -101,6 +101,8 @@ class LoginSession extends DataObject
 
     private static bool $anonymize_ip = false;
 
+    private static bool $must_use_primary_db = true;
```

Comment:
Why? Add a code comment so it's clear why we're setting this to true.

Reply:
Updated

Reply:
The comment explains what this config property does - but still doesn't say why we need it to be true for this dataobject specifically. I assume it's something to do with ensuring session data doesn't go out of sync?

Reply:
Updated